src/tools/cephfs/MDSUtility.cc

```diff
@@ -156,8 +156,7 @@ void MDSUtility::handle_fs_map(MFSMap* m)
 }
 
 
-bool MDSUtility::ms_get_authorizer(int dest_type, AuthAuthorizer **authorizer,
-                         bool force_new)
+bool MDSUtility::ms_get_authorizer(int dest_type, AuthAuthorizer **authorizer)
```

Comment:
You forgot to remove the `if (force_new)` block a few lines below.

Reply:
+1